python/dgl/graphbolt/minibatch_sampler.py

```diff
@@ -108,11 +100,72 @@ class MinibatchSampler(IterDataPipe):
     >>> data_pipe = data_pipe.map(add_one)
     >>> list(data_pipe)
     [tensor([1, 2, 3, 4]), tensor([5, 6, 7, 8]), tensor([ 9, 10])]
+
+    7. Heterogeneous node/edge IDs.
+    >>> ids = {
+    ...     ("user", "like", "item"): gb.ItemSet(torch.arange(0, 5)),
+    ...     ("user", "follow", "user"): gb.ItemSet(torch.arange(5, 10)),
+    ... }
+    >>> item_set = gb.DictItemSet(ids)
+    >>> minibatch_sampler = gb.MinibatchSampler(item_set, 4)
+    >>> list(minibatch_sampler)
+    [{('user', 'like', 'item'): tensor([0, 1, 2, 3])},
+    {('user', 'follow', 'user'): tensor([5, 6, 7]),
+        ('user', 'like', 'item'): tensor([4])},
```

Comment:
Shouldn't it be like this?
```suggestion
    {('user', 'like', 'item'): tensor([4])},
        ('user', 'follow', 'user'): tensor([5, 6, 7]),
```

Reply:
No. `('user', 'follow', 'user')` is iterated and placed to the mini-batch first, then `('user', 'like', 'item')`. But anyway, `dict` is not ordered.